app/assets/stylesheets/variables/_bootstrap.scss

```diff
@@ -843,7 +844,7 @@ $btn-primary-border:                $link-color !default;
 // $kbd-color:                   #fff
 // $kbd-bg:                      #333
 
-// $pre-bg:                      #f5f5f5
+$pre-bg:                         #f5f5f5;
```

Comment:
Why are these vars being uncommented even though the default values aren't changing...